src/app/viewChooser/ViewChooserAccordion.tsx

```diff
@@ -15,11 +15,13 @@ export interface IViewChooserAccordionProps {
    * Open view callback
    */
   onSelectedView: (view: IViewPluginDesc) => void;
+  workbenchName: string;
 
   /**
    * Currently open view
    */
   selectedView?: IViewPluginDesc;
+  isTransitionActive: boolean;
```

Comment:
```suggestion
  isWorkbenchTransitionDisabled: boolean;
```

Why not using the negative state, since you are using in the context of disabling it and inversing the flag in line 110 (`!isTransitionActive`) anyway?

Reply:
Done